cluster/addons/addon-manager/kube-addons.sh

```diff
@@ -116,10 +118,10 @@ function generate_prune_whitelist_flags() {
 # besides the default ones.
 extra_prune_whitelist=
 if [ -n "${KUBECTL_EXTRA_PRUNE_WHITELIST:-}" ]; then
-  extra_prune_whitelist=( ${KUBECTL_EXTRA_PRUNE_WHITELIST:-} )
+  extra_prune_whitelist=( "${KUBECTL_EXTRA_PRUNE_WHITELIST:-}" )
```

Comment:
This is wrong.  It will produce an array with one element.

```
$ KUBECTL_EXTRA_PRUNE_WHITELIST="foo bar bat"
$ extra_prune_whitelist=( "${KUBECTL_EXTRA_PRUNE_WHITELIST:-}" )
$ declare -p extra_prune_whitelist
declare -a extra_prune_whitelist=([0]="foo bar bat")

$ extra_prune_whitelist=( ${KUBECTL_EXTRA_PRUNE_WHITELIST:-} )
$ declare -p extra_prune_whitelist
declare -a extra_prune_whitelist=([0]="foo" [1]="bar" [2]="bat")
```